ocs_ci/utility/localstorage.py

```diff
@@ -169,8 +171,29 @@ def get_lso_channel():
         str: local storage operator channel
 
     """
-    channel = get_ocp_version()
-    # Workaround for https://github.com/red-hat-storage/ocs-ci/issues/2324
-    if channel == '4.5' or '4.6':
-        channel = '4.4'
-    return channel
+    ocp_version = get_ocp_version()
+    # Retrieve available channels for LSO
+    cmd = (
```

Comment:
Why not use PackageManifest object https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/packagemanifest.py ?

Reply:
Good call! I totally forgot we had this class.